### PR TITLE
docs: reflect the release version 0.5.0

### PR DESCRIPTION
**Pull request by haosanzi**
Signed-off-by: Shirong Hao <shirong@linux.alibaba.com>

### CODE REVIEW

File: docs/Running_Occlum_with_Docker_and_OCI_Runtime_rune.md [hunk 1]
```diff
@@ -38,7 +38,8 @@ This user guide provides the steps to run Occlum with Docker and OCI Runtime `ru
 
     2. Install the RPM packages.
     ```shell
-    yum install rune occlum-pal
+    yum install rune occlum
```
Comment (jiazhang0) [inline docs/Running_Occlum_with_Docker_and_OCI_Runtime_rune.md:41]: Plz add the "-y" for all yum install commands.

File: docs/Running_Occlum_with_Docker_and_OCI_Runtime_rune.md [hunk 2]
```diff
@@ -55,39 +56,25 @@ This user guide provides the steps to run Occlum with Docker and OCI Runtime `ru
     3. Update the apt and install the packages.
     ```shell
     apt-get update
-    apt-get install rune occlum-pal
+    apt-get install rune occlum
+    source /etc/profile
```
Comment (jiazhang0) [inline docs/Running_Occlum_with_Docker_and_OCI_Runtime_rune.md:60]: Why source this file?

Comment (haosanzi) [reply docs/Running_Occlum_with_Docker_and_OCI_Runtime_rune.md:60]: To make the new installed occlum binaries and libraries work

File: docs/Running_Occlum_with_Docker_and_OCI_Runtime_rune.md [hunk 3]
```diff
@@ -55,39 +56,25 @@ This user guide provides the steps to run Occlum with Docker and OCI Runtime `ru
     3. Update the apt and install the packages.
     ```shell
     apt-get update
-    apt-get install rune occlum-pal
+    apt-get install rune occlum
```
Comment (jiazhang0) [inline docs/Running_Occlum_with_Docker_and_OCI_Runtime_rune.md:59]: Add -y for all apt-get install.

**Review by jiazhang0 — CHANGES_REQUESTED** [review]

File: docs/Running_Occlum_with_Docker_and_OCI_Runtime_rune.md (hunk 2 above)
Comment (jiazhang0) [reply docs/Running_Occlum_with_Docker_and_OCI_Runtime_rune.md:60]: That is OK.

**Review by jiazhang0 — APPROVED** [review]

**Pull request merged**